package.json

```diff
@@ -60,15 +60,18 @@
     "async": "1.5.2",
     "bootstrap": "3.3.6",
     "codecov": "1.0.1",
+    "codelyzer": "0.0.19",
     "compression-webpack-plugin": "0.3.1",
     "conventional-changelog-cli": "1.2.0",
     "conventional-github-releaser": "1.1.2",
     "copy-webpack-plugin": "2.1.3",
     "cpy-cli": "1.0.1",
+    "del": "^2.2.0",
     "del-cli": "0.2.0",
     "es6-promise": "3.1.2",
     "es6-shim": "0.35.0",
     "es7-reflect-metadata": "1.6.0",
+    "eslint": "^2.10.2",
```

Comment:
version of codelyzer and eslint is controlled by `eslint-config-valorsoft` package,
please remove